### PR TITLE
sispmctl: Update to 4.0

### PR DESCRIPTION
**Pull request by neheb**
Switched from git to tarball. There seems to be no activity since release
4.0.

Removed unneeded patches.

Added a library package since it seems to be required now.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @rkunze 
Compile tested: mvebu

### CODE REVIEW

File: utils/sispmctl/Makefile [hunk 1]
```diff
@@ -49,9 +54,15 @@ CONFIGURE_ARGS += \
...
 	--disable-dependency-tracking
 
+define Package/libsispmctl/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/lib$(PKG_NAME)* $(1)/usr/lib/
+endef
+
 define Package/sispmctl/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
```
Comment (yousong) [inline utils/sispmctl/Makefile]: Please just use the actual name here.  No good with the variable ref

Comment (neheb) [reply utils/sispmctl/Makefile]: Done

File: utils/sispmctl/Makefile [hunk 2]
```diff
@@ -49,9 +54,15 @@ CONFIGURE_ARGS += \
...
 	--disable-dependency-tracking
 
+define Package/libsispmctl/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/libsispmctl* $(1)/usr/lib/
+endef
+
 define Package/sispmctl/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
```
Comment (hnyman) [inline utils/sispmctl/Makefile]: On line 64 there is still the /usr/bin/$(PKG_NAME) that looks confusing.

**Comment by neheb** [comment]
Yeah I can't read. Should be fine now.

**Pull request merged**